ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/view/service/ViewServiceImpl.java

```diff
@@ -43,10 +49,25 @@ public void addView(ViewSpecification viewSpecification) {
 			throw new ExistingResourceException(VIEW_TYPE, viewSpecification.getName().asString());
 		}
 
+		checkViewForDuplicateRetentionPolicies(viewSpecification);
+
 		eventPublisher.publishEvent(new ViewAddedEvent(viewSpecification));
 		viewRepository.saveView(viewSpecification);
 	}
 
+	private void checkViewForDuplicateRetentionPolicies(ViewSpecification viewSpecification) {
+		viewSpecification.getRetentionConfigs().stream()
+				.map(retentionPolicy -> retentionPolicy.listStatements(null, RdfConstants.RDF_SYNTAX_TYPE, (Resource) null).nextStatement())
```

Comment:
Can you use `retentionPolicy.listObjectsOfProperty` instead of listing the statements and then getting the objects?